pydata_sphinx_theme/theme.conf

```diff
@@ -31,4 +31,7 @@ navbar_start = navbar-logo.html
 navbar_center = navbar-nav.html
 navbar_end = navbar-icon-links.html
 footer_items = copyright.html, sphinx-version.html
-page_sidebar_items = page-toc.html, edit-this-page.html
\ No newline at end of file
+page_sidebar_items = page-toc.html, edit-this-page.html
+switcher_json_url =
```

Comment:
Since we're asking people to use `html_context` above, I don't believe that we'd need to define these variables in the theme options, though I think probably the right thing to do is recommend people put them in `html_theme_options` instead as this configuration would suggest